src/redisai.c

```diff
@@ -1528,6 +1543,16 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     else if (strcasecmp(key, "ONNX") == 0) {
       ret = RAI_LoadBackend(ctx, RAI_BACKEND_ONNXRUNTIME, val);
     }
+    // enable configuring the main thread to create a fixed number of worker threads up front per device.
+    // by default we'll use 1
+    else if (strcasecmp(key, "PER-QUEUE-THREADS") == 0) {
```

Comment:
`PER-QUEUE-THREADS` I'd rename this to `THREADS_PER_QUEUE`

Reply:
done